### PR TITLE
Bug/infinite percent

### PR DESCRIPTION
**Pull request by Tulun**
**Summary**
Display bug in tvl.
https://app.shortcut.com/uma-project/story/2361/fix-change-in-umaverse

**Details**
Error when calculating percent when x1 (basic growth formula) was 0. Would show infinity. Added a utility function to handle this edge case.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [x]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "start": "next start",
     "type-check": "tsc",
     "prepare": "husky install",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "test": "jest --watch"
```
Comment (daywiss) [inline package.json:11]: i think we want `test` as a convention that could be run in ci, so probably dont want it to be in watch mode here.
maybe add a separate `test:watch`

File: utils/tests/format.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```
Comment (daywiss) [inline utils/tests/format.test.js:1]: imo i would put this file in the same directory as `utils/format`, since its already differentiated as `format.test`

File: package.json (hunk 1 above)
Comment (Tulun) [reply package.json:11]: Good call.

File: utils/tests/format.test.js (hunk 2 above)
Comment (Tulun) [reply utils/tests/format.test.js:1]: It's generally good practice, given how many JS files there are already in this folder, to group the tests together so it's a bit cleaner in our code editor.

File: pages/index.tsx [hunk 3]
```diff
@@ -96,17 +97,13 @@ const IndexPage: React.FC<InferGetStaticPropsType<typeof getStaticProps>> = ({
...
-              formatWeiString(ydayTvl)) *
-              1000
-          ) / 10
-        : 0;
+      return formatTvlChange(ydayTvl, totalTvl!);
     },
     { enabled: Boolean(totalTvl) }
   );
 
+  console.log("all synths", allSynths);
```
Comment (daywiss) [inline pages/index.tsx:105]: remove log

**Review by daywiss — APPROVED** [review]

**Review by Gamaranto — APPROVED** [review]
